src/main/java/com/stripe/param/SourceCreateParams.java

```diff
@@ -0,0 +1,1105 @@
+        String userAgent;
+
+        private Online(Long date, String ip, String userAgent) {
+          this.date = date;
+          this.ip = ip;
+          this.userAgent = userAgent;
+        }
+
+        public static Builder builder() {
+          return new com.stripe.param.SourceCreateParams.Mandate.Acceptance.Online.Builder();
+        }
+
+        public static class Builder {
+          private Long date;
+
+          private String ip;
+
+          private String userAgent;
+
+          /** Finalize and obtain parameter instance from this builder. */
+          public Online build() {
+            return new Online(this.date, this.ip, this.userAgent);
+          }
+
+          /** The unix timestamp the mandate was accepted or refused at by the customer. */
+          public Builder setDate(Long date) {
+            this.date = date;
+            return this;
+          }
+
+          /** The IP address from which the mandate was accepted or refused by the customer. */
+          public Builder setIp(String ip) {
+            this.ip = ip;
+            return this;
+          }
+
+          /**
+           * The user agent of the browser from which the mandate was accepted or refused by the
+           * customer.
+           */
+          public Builder setUserAgent(String userAgent) {
+            this.userAgent = userAgent;
+            return this;
+          }
+        }
+      }
+
+      public enum Status implements ApiRequestParams.Enum {
+        @SerializedName("accepted")
+        ACCEPTED("accepted"),
+
+        @SerializedName("pending")
+        PENDING("pending"),
+
+        @SerializedName("refused")
+        REFUSED("refused"),
+
+        @SerializedName("revoked")
+        REVOKED("revoked");
+
+        @Getter private final String value;
+
+        Status(String value) {
+          this.value = value;
+        }
+      }
+
+      public enum Type implements ApiRequestParams.Enum {
+        @SerializedName("offline")
+        OFFLINE("offline"),
+
+        @SerializedName("online")
+        ONLINE("online");
+
+        @Getter private final String value;
+
+        Type(String value) {
+          this.value = value;
+        }
+      }
+    }
+
+    public enum Empty implements ApiRequestParams.Enum {
+      @SerializedName("")
+      EMPTY("");
+
+      @Getter private final String value;
+
+      Empty(String value) {
+        this.value = value;
+      }
+    }
+
+    public enum Interval implements ApiRequestParams.Enum {
+      @SerializedName("one_time")
+      ONE_TIME("one_time"),
+
+      @SerializedName("scheduled")
+      SCHEDULED("scheduled"),
+
+      @SerializedName("variable")
+      VARIABLE("variable");
+
+      @Getter private final String value;
+
+      Interval(String value) {
+        this.value = value;
+      }
+    }
+
+    public enum NotificationMethod implements ApiRequestParams.Enum {
+      @SerializedName("deprecated_none")
+      DEPRECATED_NONE("deprecated_none"),
```

Comment:
Can you clarify what this one even is?

Reply:
This goes away when we decide to undoc mandate